### PR TITLE
Remove missing link from Modal doc

### PR DESCRIPTION
**Pull request by AgtLucas**
Is okay to remove this link? It seems that we don't have the Navigator Comparison anymore.

### CODE REVIEW

File: Libraries/Modal/Modal.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const RCTModalHostView = requireNativeComponent('RCTModalHostView', null);
  * The Modal component is a simple way to present content above an enclosing view.
  *
  * _Note: If you need more control over how to present modals over the rest of your app,
- * then consider using a top-level Navigator. Go [here](/react-native/docs/navigator-comparison.html) to compare navigation options._
+ * then consider using a top-level Navigator._
```
Comment (mmmulani) [inline Libraries/Modal/Modal.js]: cool, could you also remove the trailing underscore?

Comment (AgtLucas) [inline Libraries/Modal/Modal.js]: @mmmulani If I do that, then we lose the italicized text. :)

Comment (mmmulani) [inline Libraries/Modal/Modal.js]: ahh! didn't notice that :)

**Comment by mmmulani** [comment]
@facebook-github-bot shipit

File: Libraries/Modal/Modal.js (hunk 1 above)
Comment (AgtLucas) [inline Libraries/Modal/Modal.js]: :)

**Pull request closed without merging**